sycamore/utils/extract_json.py

```diff
@@ -0,0 +1,11 @@
+import re
+import json
+
+
+def extract_json(payload):
+    pattern = r"```json([\s\S]*?)```"
```

Comment:
Is the fact that this is wrapped in ```json()``` specific to OpenAI? or is it coming from somewhere else (like guidance)? If we wanted to support a different llm at some point (say Anthropic), would we need a different implementation of this method? If it's OpenAI specific we should make that clear in the name/location.